tasks/install-matlab/v1/src/script.ts

```diff
@@ -7,8 +7,9 @@ import * as path from "path";
 export async function downloadAndRunScript(platform: string, url: string, args: string | string[]) {
     const scriptPath = await toolLib.downloadTool(url);
     const bashPath = taskLib.which("bash", true);
+    const sudoPath = taskLib.which("sudo", false);
     let bash;
-    if (platform === "win32") {
+    if (!sudoPath || platform === "win32") {

```

Comment:
Is the `platform === "win32"` necessary any more with the sudo check?

Reply:
I think that should be droppable now, the only case in which there could be issues would be if the user had a custom `sudo.exe` somewhere on the path with different behavior than the Unix `sudo`. That seems pretty unlikely to me, so the check should be droppable.